### PR TITLE
Removed old products

### PR DESCRIPTION
**Pull request by stevenfeltner**

### CODE REVIEW

**Comment by netapp-jmontel** [comment]
These are major changes to the documentation which would also require changes to the navigation tree. The product managers @giladshahar and @cohenjonathan should approve first. Thanks

File: src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md [hunk 1]
```diff
@@ -17,70 +17,86 @@ The full body attribute list is available on the [Create](https://docs.spot.io/a
...
-            "cooldown": 180,
-            "resourceLimits": {
-              "maxMemoryGib": 1500,
-              "maxVCpu": 750
-            },
             "down": {
-              "evaluationPeriods": 3
+              "maxScaleDownPercentage": 50
             },
-            "headroom": {
```
Comment (giladshahar) [inline src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md:50]: @stevenfeltner why is that removed?

File: src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md [hunk 2]
```diff
@@ -17,70 +17,86 @@ The full body attribute list is available on the [Create](https://docs.spot.io/a
...
-            "isAutoConfig": false
+            "autoHeadroomPercentage": 5,
+            "isAutoConfig": true,
+            "resourceLimits": {
+              "maxMemoryGib": 100000,
+              "maxVCpu": 20000
+            }
           },
           "capacity": {
+            "target": 0,
```
Comment (giladshahar) [inline src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md:51]: @stevenfeltner shouldn't we have target=1 to get a single node provisioned upon creation?

File: src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-launch-specification.md [hunk 3]
```diff
@@ -10,7 +10,9 @@ Create an Ocean Launch Spec with the following CloudFormation template. The full
     "SpotinstOcean": {
       "Type": "Custom::oceanLaunchSpec",
       "Properties": {
-        "ServiceToken": "arn:aws:lambda:${Region}:178579023202:function:spotinst-cloudformation",
+        "ServiceToken": {
+          "Fn::Sub": "arn:aws:lambda:${AWS::Region}:178579023202:function:spotinst-cloudformation"
```
Comment (giladshahar) [inline src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-launch-specification.md:14]: @stevenfeltner does that not work without fn::sub? just out of curiosity :)

File: src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md (hunk 1 above)
Comment (stevenfeltner) [reply src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md:50]: @giladshahar This is just example I used. We can add that in if you want. My main goal was to fix all the CF errors. The current example will not work.

File: src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md (hunk 2 above)
Comment (stevenfeltner) [reply src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md:51]: @giladshahar Sure we can, again this was just an example.

File: src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-launch-specification.md (hunk 3 above)
Comment (stevenfeltner) [reply src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-launch-specification.md:14]: @giladshahar Nope, I was not able to get that to work with out the Fn::Sub. :(

**Comment by stevenfeltner** [comment]
@netapp-jmontel Are you referring to tree changes due to removing old products we no longer support? Most of Gilad's comments are in the example text. Please confirm with team these products are no longer offered by Spot and should not be mentioned in our CloudFormation documentation.

File: src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md (hunk 1 above)
Comment (giladshahar) [reply src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md:50]: got it. yeah it's not a must. users can learn about manual headroom and go back to add it if needed.

File: src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md (hunk 2 above)
Comment (giladshahar) [reply src/docs/tools-and-provisioning/cloudformation/ocean-examples/kubernetes-ocean-cluster.md:51]: yeah sure, was just wondering if customers would copy-paste the examples just to try, guess we would prefer an Ocean that will provision a node?
